extensions/amp-story-auto-analytics/0.1/test/validator-amp-story-auto-analytics.html

```diff
@@ -0,0 +1,42 @@
+<!--
```

Comment:
Could you add a comment above the line that is being validated with something like `<!-- Valid: has correct gtag attribute and value -->`.